src/SoftwareLicense.php

```diff
@@ -561,23 +561,22 @@ public function rawSearchOptions()
 
         $tab[] = [
             'id'                 => '163',
-            'table'              => Item_SoftwareLicense::getTable(),
+            'table'              => static::getTable(),
             'field'              => 'id',
             'name'               => _x('quantity', 'Number of installations'),
             'forcegroupby'       => true,
             'usehaving'          => true,
-            'datatype'           => 'count',
+            'datatype'           => 'specific',
             'massiveaction'      => false,
-            'joinparams'         => [
-                'jointype'   => 'child',
-                'beforejoin' => [
-                    'table'      => static::getTable(),
-                    'joinparams' => ['jointype' => 'child'],
-                ],
-                'condition'  => [
-                    'NEWTABLE.is_deleted'          => 0,
-                ],
-            ],
+            'computation'        => '(' .
+                '(SELECT COUNT(*) FROM ' . Item_SoftwareLicense::getTable() .
+                ' WHERE softwarelicenses_id = ' . static::getTable() . '.id AND is_deleted = 0)' .
+                ' + ' .
+                '(SELECT COUNT(*) FROM ' . SoftwareLicense_User::getTable() .
+                ' WHERE softwarelicenses_id = ' . static::getTable() . '.id)' .
+                ')',
```

Comment:
You must not use hardcoded table names in search options. When the JOIN operation is made, the table will be assigned a dynamic alias to prevent conflicts if the same table is joined multiple times with different conditions.

You must use `TABLE`, `REFTABLE` or `NEWTABLE` placeholders, depending of what you want to achieve.